static/app/views/issueDetails/utils.tsx

```diff
@@ -48,6 +48,30 @@ export function useDefaultIssueEvent() {
   const options = user ? user.options : null;
   return options?.defaultIssueEvent ?? 'recommended';
 }
+/**
+ * Combines two TagValue arrays and combines TagValue.count upon conflict
+ */
+export function mergeTagValues(
+  tagValues1: TagValue[],
+  tagValues2: TagValue[]
+): TagValue[] {
+  const tagValueCollection = tagValues1.reduce<Record<string, TagValue>>(
+    (acc, tagValue) => {
+      acc[tagValue.name] = tagValue;
+      return acc;
+    },
+    {}
+  );
+  tagValues2.forEach(tagValue => {
+    if (tagValueCollection[tagValue.name]) {
+      tagValueCollection[tagValue.name].count += tagValue.count;
```

Comment:
Tag values also have `lastSeen` which will need to be merged as well (take the more recent one)